samples/outlook/99-preview-apis/get-item-class-async.yaml

```
@@ -0,0 +1,55 @@
+order: 5
+id: outlook-get-item-class-async
+name: Get item class (Compose)
+description: Retrieves the item class property of the message or appointment being composed.
+host: OUTLOOK
+api_set:
+    Mailbox: preview
+script:
+    content: |
+        $("#get-item-class-async").click(getItemClassAsync);
+
+        function getItemClassAsync() {
+          // This snippet returns the item class property of the selected item (PR_MESSAGE_CLASS).
+          // The API call works on a message or appointment being composed and is not supported on read items.
+          Office.context.mailbox.item.getItemClassAsync((asyncResult) => {
+            if (asyncResult.status === Office.AsyncResultStatus.Failed) {
+              console.log("Action failed with error: " + asyncResult.error.message);
+              return;
+            }
+
+            console.log("ItemClass: " + asyncResult.value);
+          });
+        }
+    language: typescript
+template:
+    content: |
+        <button id="run" class="ms-Button">
+            <span class="ms-Button-label">Run</span>
+        </button>
```

```suggestion
        <section class="ms-font-m">
	    <p class="ms-font-m">This sample shows how to get the Exchange Web Services (EWS) item class of the current message or appointment being composed.</p>
	    <p><b>Required mode</b>: Message Compose, Appointment Organizer</p>
        </section>

        <section class="samples ms-font-m">
	    <h3>Try it out</h3>
	    <button id="get-item-class-async" class="ms-Button">
                <span class="ms-Button-label">Get the item class</span>
	    </button>
        </section>
```
Suggesting to add an explanation to the task pane for consistency with other samples.